fiftyone/utils/sam.py

```
@@ -411,6 +411,38 @@ def _to_abs_boxes(boxes, img_width, img_height, chunk_size=1e6):
     return boxes_xyxy
 
 
+def _to_abs_mask(mask, abs_box, img_width, img_height):
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `_to_abs_mask`

(ANN202)